package.json

```diff
@@ -10,13 +10,34 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@react-native-async-storage/async-storage": "^1.15.14",
+    "@react-native-picker/picker": "^2.2.1",
+    "@react-navigation/bottom-tabs": "^6.0.9",
+    "@react-navigation/native": "^6.0.6",
+    "@react-navigation/native-stack": "^6.2.5",
+    "axios": "^0.24.0",
+    "expo-dark-mode-switch": "^0.0.2",
+    "i18n-js": "^3.8.0",
+    "i18next": "^21.6.4",
+    "lottie-ios": "^3.2.3",
+    "lottie-react-native": "^5.0.1",
     "react": "17.0.2",
-    "react-native": "0.66.4"
+    "react-i18next": "^11.15.2",
+    "react-native": "0.66.4",
+    "react-native-dropdown-picker": "^5.2.3",
+    "react-native-localize": "^2.1.7",
+    "react-native-modal": "^13.0.0",
+    "react-native-safe-area-context": "^3.3.2",
+    "react-native-screens": "^3.10.1",
+    "react-native-shadow-2": "^6.0.0-beta.0",
```

Comment:
Shadow desteği var react native'in. Android için elevation, iOS için shadow propu var